app/root.tsx

```diff
@@ -1,22 +1,38 @@
 import {Links, LiveReload, Meta, Outlet, Scripts} from '@remix-run/react'
 import type {MetaFunction, LinksFunction} from '@remix-run/cloudflare'
-import ogImage from '~/assets/stampy-ui-preview.png'
 import styles from '~/root.css'
 
+import {useLoaderData} from '@remix-run/react'
+
 const title = 'Stampy'
-const description = 'Questions and answers about about AI Alignment'
+const description = 'AI Safety FAQ'
+const ogImage = 'https://github.com/StampyAI/StampyAIAssets/blob/main/banner/stampy-ui-preview.png'
+const twitterCreator = '@stampyai'
 export const meta: MetaFunction = () => ({
   title,
   description,
   'og:title': title,
   'og:description': description,
-  'og:image': `https://stampy-ui.aprillion.workers.dev${ogImage}`,
-  'twitter:image': 'summary_large_image',
-  'twitter:creator': '@aprillion0042',
+  'og:image': ogImage,
+  'twitter:image': ogImage,
+  'twitter:creator': twitterCreator,
 })
 export const links: LinksFunction = () => [{rel: 'stylesheet', href: styles}]
 
+export const loader = async ({request}: Parameters<LoaderFunction>[0]) => {
+  const isDomainWithLogo = request.url.match(/ui.stampy.ai/)
+  const isLogoForcedOff = request.url.match(/minLogo/)
+  const isLogoForcedOn = request.url.match(/funLogo/)
+  const minLogo = isDomainWithLogo ? !!isLogoForcedOff : !isLogoForcedOn
+
+  return {
+    minLogo,
+  }
+}
+
 export default function App() {
+  const {minLogo} = useLoaderData<ReturnType<typeof loader>>()
+
```

Comment:
This seems like a bad idea, duplicating the code to check minLogo. I couldn't figure out how to pass the value of minLogo here to the App component in index.tsx. It works for now but there's got to be a better way.

Reply:
Data from root can be passed down to routes using https://remix.run/docs/en/v1/api/remix#useoutletcontext .. I can push a commit with that change.